clusters/app.ci/cert-manager/release-controller_ingress.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    cert-manager.io/cluster-issuer: cert-issuer
+  name: release-controller
+  namespace: ci
+spec:
+  tls:
+    - secretName: release-controller-tls
+      hosts:
+        - origin-release.ci.openshift.org
```

Comment:
This is not as requested in the JIRA card. Can we do the subdomain approach instead?

Reply:
Is this mapping OK?

openshift-release.apps.ci: amd64.ocp.releases.ci.openshift.org 
openshift-release-ppc64le.apps.ci: ppc64le.ocp.releases.ci.openshift.org 
openshift-release-s390x: s390x.ocp.releases.ci.openshift.org 
origin-release.apps.ci: amd64.origin.releases.ci.openshift.org

Reply:
Yes